modules/integration/src/test/scala/scala/cli/integration/RunScalaJsTestDefinitions.scala

```diff
@@ -291,6 +291,152 @@ trait RunScalaJsTestDefinitions { _: RunTestDefinitions =>
     }
   }
 
+  test("remap imports directive") {
+    val importmapFile = "importmap.json"
+    val outDir = "out"
+    val fileName = os.rel / "run.scala"
+
+    val inputs = TestInputs(
+      fileName ->
+        s"""//> using jsEsModuleImportMap $importmapFile
+          | //> using jsModuleKind es
+          | //> using jsMode fastLinkJS
+          | //> using platform js
+          |
+           |import scala.scalajs.js
+           |import scala.scalajs.js.annotation.JSImport
+           |import scala.scalajs.js.typedarray.Float64Array
+           |
+           |object Foo {
+           |  def main(args: Array[String]): Unit = {
+           |    println(Array(-10.0, 10.0, 10).mkString(", "))
+           |    println(linspace(0, 10, 10).mkString(", "))
+           |  }
+           |}
+           |
+           |@js.native
+           |@JSImport("@stdlib/linspace", JSImport.Default)
+           |object linspace extends js.Object {
+           |  def apply(start: Double, stop: Double, num: Int): Float64Array = js.native
+           |}""".stripMargin,
+      os.rel / importmapFile -> """{"imports": {"@stdlib/linspace": "https://cdn.skypack.dev/@stdlib/linspace"}}""".stripMargin
+    )
+    inputs.fromRoot { root =>
+      val absOutDir = root / outDir
+      val outFile = absOutDir / "main.js"
+      os.makeDir.all(absOutDir)
+      os.proc(
+        TestUtil.cli,
+        "--power",
+        "package",
+        fileName,
+        "--js",
+        "--js-module-kind", "ESModule",
+        "--js-cli-version", "1.15.0.1",
```

Comment:
I think this shouldn't be needed after the rebase, `1.15.0.1` should now be used by default
```suggestion
```

Reply:
Agreed